app/views/people/show.html.haml

```diff
@@ -21,6 +21,7 @@
     %dt Role w systemie
     %dd= @person.roles.join(", ")
   = link_to "Edytuj", edit_person_path(@person), class: %w{btn btn-primary btn-sm}
+  //= link_to "Napisz wiadomość", send_email_path(), class: %w{btn btn-primary btn-sm} 
```

Comment:
Żeby to zadziałało, trzeba dodać linijkę w `config/routes.rb`:
```ruby
get 'mail/write_email', to: 'mail#write_email', as: :write_email
post 'mail/send_email', to: "mail#send_email', as: send_email
```

A link powinien posiadać metodę `write_email_path` zamiast `send_email_path()`.